src/java/test/org/apache/zookeeper/server/quorum/auth/QuorumAuthUpgradeTest.java

```diff
@@ -83,7 +83,7 @@ public void testNullAuthLearnerServer() throws Exception {
         Map<String, String> authConfigs = new HashMap<String, String>();
         authConfigs.put(QuorumAuth.QUORUM_SASL_AUTH_ENABLED, "false");
 
-        String connectStr = startQuorum(2, authConfigs, 0);
+        String connectStr = startQuorum(2, authConfigs, 0, false);
```

Comment:
I am curious why we don't need a delay here - is it because the ApacheDS is not used in this test case?